app/models/concerns/hyrax/ability.rb

```diff
@@ -231,19 +231,5 @@ def can_review_submissions?
                .where('sipity_workflow_roles.role_id' => approving_role.id).any?
         end
       end
-
-      ##
```

Comment:
this turns out to hurt more than help. We'll want to just implement the correct permissions as we go, and this obfuscates which ones are missing.